io.catenax.special_characteristics.measurement/1.0.0/Measurement.ttl

```diff
@@ -0,0 +1,83 @@
+@prefix serialPart: <urn:samm:io.catenax.serial_part:3.0.0#> .
+
+:SpecialCharacteristicMeasurement a samm:Aspect ;
+   samm:preferredName "SpecialCharacteristicMeasurment"@en ;
+   samm:description "Contains the Measurment for a special characteristic"@en ;
+   samm:properties ( serialPart:localIdentifiers bpn:bpnlProperty serialPart:customerPartId :characteristicId :results [ samm:property :measurementType; samm:optional true ] :revisionIndex ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:characteristicId a samm:Property ;
+   samm:preferredName "characteristicId"@en ;
+   samm:description "Identification of the special characteristic in in relation of the specified part"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "S1" .
+
+:results a samm:Property ;
+   samm:preferredName "measurement_results"@en ;
+   samm:description "List of measurementObjects"@en ;
+   samm:characteristic :ResultList .
+
+:measurementType a samm:Property ;
+   samm:preferredName "Measurement Type"@en ;
+   samm:description "This optional attribute hints the type of the measurement"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "simpleMeasurement" .
+
+:revisionIndex a samm:Property ;
+   samm:preferredName "Revision Index"@en ;
+   samm:description "This attribute contains the revision index of the special characteristic, it is necessary to identify a special characteristic"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "01" .
+
+:ResultList a samm-c:List ;
+   samm:preferredName "ResultList"@en ;
+   samm:description "List of result objects"@en ;
+   samm:dataType :MeasurmentObject .
+
+:MeasurmentObject a samm:Entity ;
+   samm:preferredName "Measurmentobject"@en ;
+   samm:description "This Entity represents a generic Measurment Object"@en ;
+   samm:properties ( :identifier :timestamp :value :unit :description ) .
+
+:identifier a samm:Property ;
+   samm:preferredName "Identifier"@en ;
+   samm:description "Use this Attribute to identify the measuremnt "@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "measurementPoint23" .
+
+:timestamp a samm:Property ;
+   samm:description "Timestamp when the measurement was taken"@en ;
+   samm:characteristic samm-c:Timestamp .
+
+:value a samm:Property ;
+   samm:characteristic :ValueCharacteristic ;
+   samm:exampleValue "12.09"^^xsd:float .
+
+:unit a samm:Property ;
+   samm:preferredName "Unit "@en ;
+   samm:description "Unit of the measured value"@en ;
+   samm:characteristic samm-c:UnitReference ;
+   samm:exampleValue "millimetre"^^<urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#curie> .
+
+:description a samm:Property ;
+   samm:preferredName "Description"@en ;
+   samm:description "Use this attribute to give a more detailed description to this measurement. "@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Lorem ipsum dolot sit amet, consetelur sadipscing eltir, sed diam nonumy eirmod tempr invidunt ut" .
```

Comment:
Please don't use a "lorem ipsum" example, since this will be a industry standard for special characteristics. The example should be useful to understand this attribute and therefore the whole model.

Reply:
done